### ISSUE TITLE
Uncaught TypeError: source.datafn is not a function

### ISSUE THREAD

**Issue by masterforever** [agershun/alasql#875]
Hi!
Trying to use alasql with react and webpack. 
Alasql is installed from npm.
In webpack the alasql folder is in noparse
_noParse: [/alasql/]_

In the js file where I want to call alasql I use
_let alasql = require('alasql');_

I then try to do the simplest thing and query an array of objects 
let temp = [{name: "Bob", age: 30},{name: "Alice", age: 34}];
let result = alasql("SELECT * FROM ?", [temp]);

But I get a:
_Uncaught TypeError: source.datafn is not a function_

Any idea why I get this?

**Comment by mathiasrw** [comment]
Can you do a `alasql.version` ?

**Comment by masterforever** [comment]
I can, it says 0.3.9.
Everything works now though. Since it didn't work the first time I had a brake point set to the alasql code, in chrome. When I tried different alasql queries in the chrome console I got the error above. But when I run the code normaly everything works fine. Any Idea how to get it work in the console? I do a lot of testing with the alasql calls and to run all the code over and over again just to test different queries is time consuming.
